userinfo-api-faq.md

```diff
@@ -53,6 +73,19 @@ payment session, we keep this information accessible for the merchant for the
 next 168 hours, even though the user revokes the consent in this period.
 Revoking consents will immediately affect future login and payment sessions.
 
+## How can I get updated information, like addresses, for a user?
+
+When the user consents to sharing information with the merchant, the merchant
```

Comment:
I think it would be clearer if we split this into Login and Payment. It is confusing when we mix the differences between using those two APIs.

Reply:
Ok, good. I will remove the Login part from Userinfo and add an item in the Login FAQ.

Reply:
Done.